src/controllers/country.js

```diff
@@ -0,0 +1,179 @@
+import Admin from "../services/AdminServices/countryService";
+import { validation } from "../validation/countryValidation";
+import db from "../models";
+
+const countriesAttributes = [
+  "id",
+  "nameOfCountry",
+  "gallery",
+  "capital",
+  "population",
+  "officialLanguage",
+  "region",
+  "currency",
+];
+
+const getOneCountry = async id => {
```

Comment:
code climate was complaining about duplication of code because there are two routers where i needed to get a country from the db so i decided to create a function where i can just pass the id and get the country